kuma/settings/content_experiments.json

```diff
@@ -1,2 +1,85 @@
 [
+    {
+        "id": "experiment-interactive-editor",
+        "ga_name": "interactive-editor",
+        "param": "v",
+        "pages": [
+            {
+                "locale": "en-US",
+                "slug": "docs/Web/JavaScript/Reference/Global_Objects/Array/push",
```

Comment:
The ``slug`` should be ``Web/JavaScript/Reference/Global_Objects/Array/push``, the ``docs`` is implied.  Same with other ``slug`` values in the JSON.